docs/dev/docker-hub.md

```diff
@@ -26,19 +26,19 @@ such that limiting the number of available cores (with `--smp 1`) is
 the simplest way to go.
 
 Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
-On many non-production systems, it will be equal to 65K. The formula
-to calculate the proper value is:
+On many non-production systems, it will be equal to 64K. The formula
+to calculate the proper value depends on the current value of `/proc/sys/fs/aio-nr`
+and also on the number of cores to be used by ScyllaDB:
 
-    Available AIO on the system - (request AIO per-cpu * ncpus) =
-    aio_max_nr - aio_nr < (reactor::max_aio + detect_aio_poll + reactor_backend_aio::max_polls) * cpu_cores =
-    aio_max_nr - aio_nr < (1024 + 2 + 10000) * cpu_cores =
-    aio_max_nr - aio_nr < 11026 * cpu_cores
+    Available AIO on the system >= (request AIO per-cpu) * ncpus
 
-    where
+meaning
 
-    reactor::max_aio = max_aio_per_queue * max_queues,
-    max_aio_per_queue = 128,
-    max_queues = 8.
+    aio_max_nr - aio_nr >= (storage_iocbs + preempt_iocbs + network_iocbs) * ncpus
+                                     1024               2           50000
+meaning
```

Comment:
probably could change to "So the calculation would be:" so less repeating, and easier to digest.

Reply:
OK, I'll refine this in v2.